azavea/raster-foundry
# Update dependency pinning for frontend

## Overview

This fixes an issue caused by using `^` for dependencies. This commit
changes as many dependencies as possible without breaking the
application. The dependencies that caused issues were the babel
ones so those ones remained unchanged.

### Checklist

~~- [ ] Styleguide updated, if necessary~~
~~- [ ] Swagger specification updated, if necessary~~

## Testing Instructions

 * Clean dependencies (`rm -rf ./node_modules`) on `develop`, run `npm install` and, load the application
 * Observe that filters and other functionality is broken
 * Switch to PR branch, clean dependencies, run `npm install`, and load the application
 * Observe that application works correctly again
